src/middlewares/backup_auth.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable prefer-destructuring */
+import jwt from 'jsonwebtoken';
+
+const backup_auth = {
+  generateToken(user) {
+    return jwt.sign({
+      id: user.id,
+      role: user.role,
+    }, process.env.SECRET, { expiresIn: '7d' });
+  },
+
+  verifyToken(token) {
+    let decoded = {};
+    try {
+      decoded.payload = jwt.verify(token, process.env.SECRET);
+    } catch (error) {
+      decoded = { error: error.message };
+    }
+    return decoded;
+  },
+
+  async verifyUserToken(req, res, next) {
+    try {
+      const token = req.headers.authorization.split(' ')[1] || req.headers['x-access-token']
```

Comment:
Again, for the sake of consistency, I suggest we stick to `req.headers.authorization`

Reply:
See above comment.

Reply:
see above comment also. lol

Reply:
see above comment again lol